subsys/net/lib/ftp_client/src/ftp_commands.h

```diff
@@ -120,7 +68,7 @@ extern "C" {
 /* Remove a directory */
 #define CMD_RMD		"RMD %s\r\n"
 /* Remove a directory tree */
-#define CMD_RMDA	"RMDA\r\n"
+//#define CMD_RMDA	"RMDA\r\n"
```

Comment:
Please remove commented code. Or use C-style comments if you want to keep it for any reason.

Reply:
deleted